python/python_fastapi_server/routers/edr.py

```diff
@@ -646,7 +686,22 @@ async def get_capabilities(collname):
     else:
         logger.info("callADAGUC by service %s", dataset)
         wms = WebMapService(dataset["service"], version="1.3.0")
-    return wms.contents
+        now = datetime.utcnow()
+        ttl = None
```

Comment:
Make a routine called convert ttl to epoch?

Reply:
Not necassary anyo\more